commcare_connect/opportunity/forms.py

```diff
@@ -334,17 +335,48 @@ def __init__(self, *args, **kwargs):
             Row(Field("name")),
             Row(Field("description")),
             Row(Field("amount")),
-            Row(Field("deliver_units")),
+            Row(Field("required_deliver_units")),
+            Row(Field("optional_deliver_units")),
             Submit(name="submit", value="Submit"),
         )
 
         choices = [(deliver_unit.id, deliver_unit.name) for deliver_unit in deliver_units]
-        self.fields["deliver_units"] = forms.MultipleChoiceField(choices=choices, widget=forms.CheckboxSelectMultiple)
+        self.fields["required_deliver_units"] = forms.MultipleChoiceField(
+            choices=choices,
+            widget=forms.CheckboxSelectMultiple,
+            help_text="All of the selected Deliver Units are required for payment accrual.",
```

Comment:
nit: ideally this help text would be in a popover or under the title but above the options. I think it looks a little odd below the options.